### ISSUE TITLE
Accordions should be keyboard accessible

### ISSUE THREAD

**Issue by shawnbot** [18F/college-choice#54]
Currently the accordion elements as specced in #49 don't pass ~~508~~ WCAG 2.0 AA muster, primarily because they don't provide keyboard accessibility. This can be achieved by replacing the `<a aria-controls="...">` elements with `<button>` elements, but that requires some CSS reworking as well.

**Comment by barkimedes** [comment]
@LisaGee I upped the priority on this because I don't know of acceptable workarounds for non-accessible elements.

**Comment by shawnbot** [comment]
The fix is, theoretically, pretty straightforward:

1. Replace instances of `a` elements in accordions with `button`s.
2. Update the CSS to style `button`s inside of accordions so that they don't look like buttons.

**Comment by shawnbot** [comment]
This appears to have been pretty easy! PR forthcoming.

**Comment by ultrasaurus** [comment]
oops -- this wasn't actually in the last release.  @shawnbot will amend release notes, re-opening

**Comment by barkimedes** [comment]
@shawnbot Where should I be able to try this out?

**Comment by abisker** [comment]
@barkimedes you can test on the dev server https://federalist.18f.gov/preview/18F/college-choice/dev/

(details in the "[college choice] pls test accessible accordian!" email)

**Comment by barkimedes** [comment]
WOOOHOO!! :us:

**Comment by barkimedes** [comment]
Splendiforous :us:
